kde-apps/dragon/dragon-9999.ebuild

```diff
@@ -16,17 +16,21 @@ SLOT="6"
 KEYWORDS=""
 IUSE=""
 
+# Upstream only supports the ffmpeg backend https://bugs.kde.org/show_bug.cgi?id=506940
+# slot op: Uses Qt6MultimediaPrivate for Qt6::QFFmpegMediaPlugin
 DEPEND="
 	>=dev-qt/qtbase-${QTMIN}:6[dbus,gui]
 	>=dev-qt/qtdeclarative-${QTMIN}:6
-	>=dev-qt/qtmultimedia-${QTMIN}:6
+	>=dev-qt/qtmultimedia-${QTMIN}:6=[ffmpeg]
 	>=kde-frameworks/kconfig-${KFMIN}:6
 	>=kde-frameworks/kcoreaddons-${KFMIN}:6
 	>=kde-frameworks/kcrash-${KFMIN}:6
 	>=kde-frameworks/ki18n-${KFMIN}:6
 	>=kde-frameworks/kio-${KFMIN}:6
 	>=kde-frameworks/kirigami-${KFMIN}:6
 	>=kde-frameworks/kservice-${KFMIN}:6
+	>=kde-frameworks/kwindowsystem-${KFMIN}:6
```

Comment:
`grep -R include.*KX` reveals `#include <KX11Extras>` so it'll also need X USEdep there. (and all older versions)